weed/filer/redis/universal_redis_store_test.go

```
@@ -0,0 +1,143 @@
+	ctx := context.Background()
+	client := redis.NewClient(&redis.Options{Addr: addr})
+	t.Cleanup(func() { client.Close() })
+	if err := client.Ping(ctx).Err(); err != nil {
+		t.Fatalf("connect to redis at %s: %v", addr, err)
+	}
+
+	store := &UniversalRedisStore{Client: client}
+
+	dir := util.FullPath(fmt.Sprintf("/redis_test_%d", time.Now().UnixNano()))
+	t.Cleanup(func() {
+		store.DeleteFolderChildren(ctx, dir)
```

golangci-lint (2.12.2)

[error] Error return value of `store.DeleteFolderChildren` is not checked

(errcheck)